tests/system.rs

```diff
@@ -66,27 +84,58 @@ fn print_mismatches(result: HashMap<String, String>) {
 static HANDLE_RESULT: &'static Fn(HashMap<String, String>) = &handle_result;
 
 pub fn idempotent_check(filename: String) -> Result<(), HashMap<String, String>> {
+    let config = get_config(&filename);
+
+    println!("config: {}", &config);
```

Comment:
I wouldn't display the config, it makes the test output really noisey. You could use `debug!` or `info!`.